### ISSUE TITLE
[FEED]Add Site Management 

### ISSUE THREAD

**Issue by mike1098** [Checkmk/ansible-collection-checkmk.general#664]
**Provide your feedback**
checkmk Ansible looks almost complete to us to be able to automate commission and decommission of remote sites in our Local Monitoring.
The thing we hardly missing is Site Management.

The site management REST endpoints allow for:

POST for creating new site configurations.
PUT for updating current site configurations.
LIST for listing all current site configurations.
GET for getting a single site configuration.
DELETE for deleting a single site configuration via its site id.
LOGIN for logging into an existing site.
LOGOUT for logging out of an existing site.

As it is in REST available it shouldn't be a big deal to add it to ansible.

**Component Name**
Component Name: site

**Ansible Version**
Not relevant

```

**Collection Version**
What ever is necessary will be installed

```
**Environment**
Checkmk 2.2.0p33.cee

**Screenshots**
We are not allowed to share our screen :-(

**Additional context**
We are open to push even with throwing in some coins ;-)

**Comment by robin-checkmk** [comment]
See the linked PR, we are in the final steps of adding the `site` module. Feel free to test the branch and let us know, what you think. :)

**Comment by mike1098** [comment]
In checkmk.general.site I have two times site_id:. On at the top and one at basic_settings. Isnt that redundant information?

**Comment by robin-checkmk** [comment]
> In checkmk.general.site I have two times site_id:. On at the top and one at basic_settings. Isnt that redundant information?

We discussed exactly that here: https://github.com/Checkmk/ansible-collection-checkmk.general/pull/654#discussion_r1727183847

**Comment by robin-checkmk** [comment]
This module has just been released with [5.3.0](https://github.com/Checkmk/ansible-collection-checkmk.general/releases/tag/v5.3.0).
